tests/test_main.py

```
@@ -0,0 +1,49 @@
+"""
+Tests for the main game loop.
+"""
+
+import unittest
+from unittest.mock import patch, MagicMock
+
+import curses
```

Ruff (0.8.2)

`curses` imported but unused

Remove unused import: `curses`

(F401)